### ISSUE TITLE
 how to rebuild the html string

### ISSUE THREAD

**Issue by purple-force** [tildeio/simple-html-tokenizer#68]
When i got data by `tokenize `, how to rebuild the html string?

**Comment by rwjblue** [comment]
I don't understand the question, can you explain what you are trying to do?

**Comment by purple-force** [comment]
I can get the structural data from html string by `tokenize`, then how to generate html string by structural data?

**Comment by Baltazore** [comment]
@rwjblue I think the question was more about how to construct HTML string from parsed tokens.
And this https://github.com/tildeio/simple-html-tokenizer/blob/v0.1.1/lib/simple-html-tokenizer/generator.js#L27 looks exactly what @purple-force needs.

**Comment by Baltazore** [comment]
Do we have generators in latest version?

**Comment by rwjblue** [comment]
No, it was removed in https://github.com/tildeio/simple-html-tokenizer/commit/01479fe883cee3ead68079ec0f576cdcfc1e96fd

**Comment by Baltazore** [comment]
hey @rwjblue thanks for answer. Any particular reason to remove this part?
I am right now trying to use this tokenizer to parse and modify given HTML snippet. Am I doing it wrong (wrong tool to use) or it is still valid idea, and I just need to implement generator of HTML from tokens tree on my own?
